src/components/fabSpeedDial/fabSpeedDial.js

```diff
@@ -199,7 +199,7 @@
           offsetDelay = index * delay;
 
         styles.opacity = ctrl.isOpen ? 1 : 0;
-        styles.transform = styles.webkitTransform = ctrl.isOpen ? 'scale(1)' : 'scale(0)';
+        styles.transform = styles.webkitTransform = ctrl.isOpen ? 'scale(1)' : 'scale(0.1)';
```

Comment:
@topherfangio - should this be `0.01` to be completely visually transparent but still support mouse clicks ?